R/gadm.R

```diff
@@ -1,32 +1,95 @@
 #' Get GADM data
 #'
-#' Download sp or sf file format from GADM
+#' Download sp or sf file format from GADM in the internal library of the
+#' package and/or a specified path and/or a temporary path.
+#'
+#' If the `path` and/or `intlib` arguments are `NULL`, it will be asked
+#' interactively to the user to decide where to store the file. The function
+#' will return a message is the file is already present in the internal library
+#' or in the specify path.
 #'
 #' @param country character string, name of the country to download
 #' @param format character string, format to downlaod, either sp or sf
 #' @param level integer between 0 and 3 or 4, level of the administrative
 #'   borders, 0 being country borders.
-#' @param file_rm boolean, if TRUE, remove the dowmloaded file.
-#'   By default, TRUE.
-#' @param path character string, name where the dowloaded file is saved
+#' @param path character string, name where the download file is also saved. By
+#'   default NULL.
+#' @param intlib boolean, if the file is saved in the internal library. By
+#'   default NULL.
```

Comment:
Please write "specifies whether the downloaded file should be saved in the library of packages." instead. What happen if `NULL`? Why not having it `TRUE` or `FALSE` by default?

Reply:
Got it! We just need to explain this in the documentation.

Reply:
I update the documentation with a graph showing that